assets/js/atomic/blocks/product-elements/average-rating/support.ts

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable @wordpress/no-unsafe-wp-apis */
+/**
+ * External dependencies
+ */
+import { isFeaturePluginBuild } from '@woocommerce/block-settings';
+
+export const supports = {
+	...( isFeaturePluginBuild() && {
+		color: {
+			text: true,
+			background: true,
+		},
+		spacing: {
+			margin: true,
+			padding: true,
+		},
+		typography: {
+			fontSize: true,
+			__experimentalFontWeight: true,
+			__experimentalSkipSerialization: true,
+		},
+		__experimentalSelector: '.wc-block-components-product-average-rating',
```

Comment:
In my opinion, we can remove the `__experimentalSelector` here. By doing so, we'll also be able to eliminate the need for the` isFeaturePluginBuild() `flag for color and spacing, as they do not utilize any experimental features. I believe we can modify it to something like this:

```tsx
export const supports = {
	color: {
		text: true,
		background: true,
	},
	spacing: {
		margin: true,
		padding: true,
	},
	...( isFeaturePluginBuild() && {
		typography: {
			fontSize: true,
			__experimentalFontWeight: true,
			__experimentalSkipSerialization: true,
		},
	} ),
};
```

What do you think?

Reply:
Got it. To be honest, I'm not a big fan of using `__experimentalSkipSerialization` unless it's necessary. I prefer relying on the returned values from [useBlockProps](https://github.com/woocommerce/woocommerce-blocks/blob/0dbdf0b16d710a7c70e39ad045fa1df60429d3bd/assets/js/atomic/blocks/product-elements/average-rating/edit.tsx#L22-L24) instead of our custom function [useStyleProps](https://github.com/woocommerce/woocommerce-blocks/blob/0dbdf0b16d710a7c70e39ad045fa1df60429d3bd/assets/js/atomic/blocks/product-elements/average-rating/block.tsx#L17).

Perhaps this could be an interesting topic for GH discussion. I'll try to create a GitHub discussion for this. In the meantime, I don't mind using `__experimentalSkipSerialization` since we are already using it in many other places.